src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java

```diff
@@ -181,9 +183,18 @@ public void onOpened()
         final Label targetLabel = findPaneOfTypeByID(LIST_ELEMENT_ID_REQUEST_LOCATION, Label.class);
         targetLabel.setLabelText(request.getRequester().getDeliveryLocation().toString());
 
-        findPaneOfTypeByID(RESOLVER, Label.class).setLabelText("Resolver: " +
-                                                                 ColonyManager.getColony(colonyId).getRequestManager().getResolverForRequest(
-                                                                   request.getToken()).getDisplayName(request.getToken()).getFormattedText());
+        try
+        {
+            findPaneOfTypeByID(RESOLVER, Label.class).setLabelText("Resolver: " +
+                    ColonyManager.getColony(colonyId).getRequestManager().getResolverForRequest(
+                            request.getToken()).getDisplayName(request.getToken()).getFormattedText());
+        }
+        catch(@SuppressWarnings(EXCEPTION_HANDLERS_SHOULD_PRESERVE_THE_ORIGINAL_EXCEPTIONS) final IllegalArgumentException e)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)